CONTRIBUTING.rst

```diff
@@ -319,27 +319,28 @@ You **must** add Labels to PRs, as these are used to automatically generate Chan
 - **Cross-compatibility:** If your PR breaks cross-compatibility with a previous
   stable release of SCT, you should add the label ``compatibility``.
 
-`Here <https://github.com/neuropoly/axondeepseg/pull/44>`_ is an example of PR 
-with proper labels and description. (#TODO: Find a better example)
+`Here <https://github.com/neuropoly/axondeepseg/pull/44>`_ is an example of PR
+with proper labels and description. (#TODO: Find a better example)
```

Comment:
Is it a good example?

Reply:
a bit old and has no label, use this instead:
https://github.com/neuropoly/axondeepseg/pull/176